src/main/java/org/myrobotlab/opencv/OpenCVFilterTracker.java

```diff
@@ -131,7 +134,7 @@ public IplImage process(IplImage image, OpenCVData data) {
         float yC = (float) (boundingBox.y() + boundingBox.height()/2);
         Point2Df center = new Point2Df(xC, yC);
         pointsToPublish.add(center);
-        data.set(pointsToPublish);
+        data.put("TrackingPoints",pointsToPublish);

```

Comment:
I'd like to see TrackingPoints pulled out as a constant at the top of the class.  eventually refactored out into an enum that defines all the keys for data.

Reply:
I would prefer this be put into a common abstract VideoData base class or interface which OpenCVData derives/implement not an opencv class

Reply:
Typically for items like this, we create an enum called FieldNames that every class references for things like this.  It helps flush out a schema.